Casks/c/calibrite-profiler.rb

```diff
@@ -0,0 +1,32 @@
+cask "calibrite-profiler" do
+  version "1.3.1"
+  sha256 "8adce527e8aeb9d271405a4c9fc865e24498494eae282f32268ed3a495982691"
+
+  url "https://github.com/LUMESCA/calibrite-profiler-releases/releases/download/v#{version}/calibrite-PROFILER-#{version}.dmg",
+      verified: "github.com/LUMESCA/calibrite-profiler-releases/"
+  name "calibrite-profiler"
+  desc "Display calibration software for Calibrite, ColorChecker and X-Rite devices"
+  homepage "https://calibrite.com/"
+
+  livecheck do
+    url "https://calibrite.com/learning-centre/calibrite-profiler-release-notes"
+    regex(%r{<strong>Calibrite PROFILER ([\d.]+)</strong>}i)
+    strategy :page_match
+  end
+
+  auto_updates true
+  depends_on macos: ">= :catalina"
```

Comment:
```suggestion
  depends_on macos: ">= :high_sierra"
```

In almost all instances, what the `info.plist` says supersedes upstream's information. It's essentially a bug on their end, and they should fix the discrepancy in their build.